src/app/login-page/login-page.component.scss

```diff
@@ -0,0 +1,38 @@
+.loginCont{
+  display: flex;
+  flex-flow: column;
+  width: 400px;
+  height: 295px;
+  margin: 0 auto;
+  border: 1px solid black;
+  background-color: rgb(238, 238, 238);
+  align-items: center;
+
```

Comment:
I see an unnecessary line here. Please delete it

Reply:
Fixed